lib/services/filters.js

```diff
@@ -16,13 +16,31 @@ function filterRefs(op) {
   return op;
 }
 
+/**
+ * Generic function for testing if op is of
+ * a certain type. Just call the function with what
+ * you're looking for and then you have a filter function.
+ *
+ * - Pages? isOpFor('pages')
+ * - User? isOpFor('user')
+ * - Component Foo? isOpFor('foo')
+ *
+ * @param  {[type]}  string [description]
+ * @return {Boolean}        [description]
```

Comment:
This doesn't return a boolean. Also might make sense to rename this fnc to something like `createOpFilter` to distinguish it from the other exports of this module